tools/gitver.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+#****************************************************************************
+# tools/gitversion.sh
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+#****************************************************************************
+
+# This script will attempt to generate a semver version based on the git tags
+# and the supplied target version as either major.minor or major.minor.patch
+# If there is not yet a tag for Version 0.1 and 0.1 is supplied it will
+# return the version 0.1.0 If a tag exists of 0.1.2-RC1 it would return
+# version 0.1.2 to standard out if 0.1 or 0.1.2 is supplied
+#
+# Example usage:
+# $ ./gitver.sh 0.1
+#   0.1.2
```

Comment:
Could you please add this part to the tools/README.txt file.

Reply:
@Ouss4 Done.   If we add this to the release branch I think we will start generating the release tarballs.